client/my-sites/pages/page-list.jsx

```diff
@@ -177,29 +178,29 @@ const Pages = localize( React.createClass( {
 			);
 		}
 
-		const { site, siteId, status = 'published' } = this.props;
+		const { site, siteId } = this.props;
```

Comment:
NIT: I think it would be clearer to unify the multiple `this.props` destructuring assignments.

Reply:
Good catch on that, thanks! I agree. Do you have a recommendation for destructuring `query` at [line 165](https://github.com/Automattic/wp-calypso/pull/18345/files/312160586e17781c85727f59a30671833a826bf6#diff-602dc25e101e23d58d5413716a4074e9R165)? Should I also add `query` to the first assignment and then destructure `query` in the next line rather than `this.props.query`?